introduction.md

```diff
@@ -0,0 +1 @@
+My name is Moises Arcos and I'm Software Engineer. I'm working as a GIS developer at Emergya in Seville and I participate in a local investigation group call it "Geoinquietos". The aim of this group is divulgate GIS knowledge and promote event for the citizens.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.